music21/roman.py

```diff
@@ -628,105 +628,47 @@ def correctRNAlterationForMinor(figureTuple, keyObj):
     return FigureTuple(figureTuple.aboveBass, alter, rootAlterationString)
 
 
-def romanNumeralFromChord(chordObj,
+def romanNumeralFromChord(chordObj: chord.Chord,
                           keyObj: Union[key.Key, str] = None,
-                          preferSecondaryDominants=False):
+                          preferSecondaryDominants: bool = True):
     # noinspection PyShadowingNames
+    # TODO: preferSecondaryDominants option
     '''
-    Takes a chord object and returns an appropriate chord name.  If keyObj is
-    omitted, the root of the chord is considered the key (if the chord has a
-    major third, it's major; otherwise it's minor).  preferSecondaryDominants does not currently
-    do anything.
+    Takes a chord object (chordObj) and returns a Roman numeral.
```

Comment:
as long as we are cleaning up, should say:  
```
Takes a :class:`~music21.chord.Chord` object and an optional :class:`~music21.key.Key` object (or a string representing a key) and returns a :class:`~music21.roman.RomanNumeral` object.
```

I don't know what "are key-relative" means.  And this doctest removes the instructioin that "preferSecondaryDominants does not currently do anything".